ister_gui.py

```diff
@@ -2733,7 +2742,9 @@ def __init__(self, *args, **kwargs):
     def _init_actions(self):
         # auto install assumed
         keyboard_select = KeyboardSelection()
-        network_requirements = NetworkRequirements(1, 6)
+        network_requirements = NetworkRequirements(1, 6,
+                                                   self.args['contenturl'],
+                                                   self.args['versionurl'])
```

Comment:
I make an image using ister
```
python3 ister.py -t installer-config.json
```

I load up the local GUI and ister changes
```
./update_installer.sh installer.img
```

I run the instance with qemu
```
sudo qemu-system-x86_64 -enable-kvm -m 1024 -vnc 0.0.0.0:0 -cpu host -drive file=installer.img,if=virtio,aio=threads -net nic,model=virtio -net user,hostfwd=tcp::2233-:22 -smp 2 -bios <path to bios file>
```

Then I just use vncviewer.

I have that qemu command mapped to a bash function. If I make changes to the gui I just run `./update_installer.sh installer.img` again.